### DISCUSSION TITLE
Is django-cms-installer deprecated?

### DISCUSSION THREAD

**Discussion by TiredFingers** [django-cms/django-cms#7159] [General]
If I'm not mistake - installer not supported. Should we remove references to installer from the docs?

**Comment by crydotsnake** [comment]
I would not say that it is no longer supported. It's just that the djangoCMS installer still installs djangoCMS version 3.8 with the Treebeard issue. The installer has not been updated for a long time.

But with the docs part I am on your side. We could talk about that.

**Reply by crydotsnake** [reply]
In my opinion we could also think about adding the djangoCMS installer into our organization and manage it independently. Of course, we should first ask the main developer for his approval.

**Reply by mazulo** [reply]
I kindly asked in this [thread](https://github.com/nephila/djangocms-installer/issues/388#issuecomment-1158826766) how's the project going. I think the idea of moving this project to the django-cms organization is a great idea.

**Comment by macolo** [comment]
Yes, we should remove the references to the installer from the docs.

**Comment by fsbraun** [comment]
django CMS 4.1+ comes with a `djangocms` command that replaces the installer.
